pkg/e2etemplates/helper_test.go

```diff
@@ -0,0 +1,108 @@
+package e2etemplates
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"testing"
+)
+
```

Comment:
Here's the `diff` function which makes it easier to spot rendered values differences.

Reply:
Isn't there some diff lib we can just use?